pkg/internal/checkup/executor/executor.go

```diff
@@ -34,21 +34,21 @@ type vmiSerialConsoleClient interface {
 }
 
 type Executor struct {
-	client      vmiSerialConsoleClient
-	namespace   string
-	vmiUsername string
-	vmiPassword string
+	vmiSerialClient vmiSerialConsoleClient
```

Comment:
This change could be a part of the previous commit.

Reply:
DONE